### PR TITLE
system-variables: add init connect

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5755
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by xhebox — APPROVED** [review]
LGTM

File: system-variables.md [hunk 1]
```diff
@@ -104,6 +104,12 @@ mysql> SELECT * FROM t1;
 - 默认值：（系统主机名）
 - 这个变量一个只读变量，表示 TiDB server 的主机名。
 
+### `init_connect`
+
+- 作用域：GLOBAL
+- 默认值：""
+- `init_connect` 特性允许首次连接到 TiDB 服务器时自动执行 SQL 语句。如果你有 `CONNECTION_ADMIN` 或者 `SUPER` 权限，此条 `init_connect` 语句将不会被执行。如果 `init_connect` 语句报错，你的用户连接将被终止。
```
Comment (TomShawn) [inline system-variables.md:111]: @xhebox PTAL whether this comment is correct. Thanks!
```suggestion
- 用户首次连接到 TiDB 服务器时，`init_connect` 特性允许 TiDB 自动执行一条 SQL 语句。如果你有 `CONNECTION_ADMIN` 或者 `SUPER` 权限，此条 SQL 语句将不会被自动执行。如果此条语句执行报错，你的用户连接将被终止。
```

Comment (xhebox) [reply system-variables.md:111]: I think it is not limited to only one stmt.

Comment (TomShawn) [reply system-variables.md]: @xhebox I've changed my comment according to your suggestion. PTAL again, thanks!

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**